Cvdm.ErrorHandling/AsyncResultBuilder.fs

```diff
@@ -29,11 +29,11 @@ type AsyncResultBuilder() =
   member this.Bind (result: Result<'a, 'c>, binder: 'a -> Async<Result<'b, 'c>>) : Async<Result<'b, 'c>> =
     this.Bind(this.ReturnFrom result, binder)
 
-  member __.TryWith (body, handler) =
-    try body() with e -> handler e
+  member __.TryWith ((body : (Unit -> Async<Result<'a, 'b>>)), handler : (System.Exception -> Async<Result<'a, 'b>>)) : Async<Result<'a, 'b>> =
```

Comment:
Stick to existing style: Remove parentheses around type annotations (not needed for member definitions), and no space before the colons.